proxy_test.go

```diff
@@ -51,6 +51,9 @@ var goodCfg = &config.Config{
 			ToUser:    "web",
 		},
 	},
+	ParamGroups: []config.ParamGroup{
```

Comment:
I added it to make sure that we can test instantiation of params registry while creating proxy